### ISSUE TITLE
see if installer can copy music tracks from re-release

### ISSUE THREAD

**Issue by neogeographica** [neogeographica/quakestarter#63]
I think Bethesda and Steam have the re-release at this point, and GOG may get it as well.

If the music tracks from those installations can work fine with community Quake engines, it would be appropriate to retire the soundtrack-download code and only get the soundtrack from local file copy.

**Comment by GenericJohnDoeNO0** [comment]
The soundtrack files from the current remaster are not desireable, the devs did a f..k-up while ripping the tracks and put a de-emphasize on them. The NIN-Soundtrack of id1 sounds now more mute and muffled than the original and desireable version.

But i currently do not now if the quaddicted and quaketastic files got the same "good meant" treatment, too.
I did mount the disimages which are provided by the gog version of "Quake: The Offering" and did rip the tracks there and converted them without de-emphasizing to *.ogg. They sound exactly in my quakespasm spiked installation as they should.
(Trent Reznor / NiN used that sharp, metallic sound as a element of style and a rehearing of my original quake cd in a stereo with headphones did confirmed my oppinion.)

Tldr:

No, remaster does sound wrong because of reformating errors of the devs.

**Comment by neogeographica** [comment]
Thanks for weighing in. FWIW, the ones that quakestarter will DL from quaddicted also honor the pre-emphasis marker, although I haven't heard the tracks from the remaster yet so I don't know overall which set might sound better or worse.

I think this really is a change I have to do though, now that the tracks are officially provided. Folks who want to rip their own still can of course.

**Comment by GenericJohnDoeNO0** [comment]
Well - in contrast to my own rip - the quaddicted files have gotten the pre-emphasis and the files from the remaster double that. Both are very muffled, like through an old radio. But the remaster ones are far worse, maybe the publisher grapped the quaddicted files and did the cursed pre-emphasis again....

Best audio solution is to get hands on the original cd or gogs release of Quake the offering, rip the audio tracks and convert them lossless to ogg. 

The sharp and metallic emphasis is and was for ever a part of Reznors style. Every NiN-Album i have sounds similar sharp and harsh - and this style fits.

**Comment by neogeographica** [comment]
The Bethesda launcher is [being sunsetted in April](https://bethesda.net/en/article/2RXxG1y000NWupPalzLblG/sunsetting-the-bethesda-net-launcher-and-migrating-to-steam) so I might not need to worry about that? However it does have the music folders in there now. So I could look for those. Also see issue #69 though.

For the Steam version I should look under \<steam path stuff>/Quake/rerelease followed by id1, hipnotic, or rogue then the music folder.

Xbox Game Pass is another place it could look for the music (although not the original pak files I don't think). By default it ends up at XboxGames/QUAKE/Content followed by id1, hipnotic, or rogue then the music folder. I'm pretty sure the install paths/preferences for Game Pass are stored somewhere in the protected WindowsApps folder, so I can't detect those.

Still no GOG update.

If I did do reliable pak hash detection (issue #43) then _install_quakefiles.cmd could always just look in all possible locations, including rerelease locations.

If not...

One way to attack this would be to add an optional argument to _install_quakefiles.cmd that indicates whether to look in rerelease locations. When looking for pak files a caller would not set that arg, but when looking for music dirs would invoke it once without the arg and (if necessary) once with the arg.

**Comment by neogeographica** [comment]
Starting work on this with https://github.com/neogeographica/quakestarter/commit/bfac79d3189b20886b928465ae1651d345e15682 .
